### PR TITLE
Cmd line argument to set log level

### LINKED ISSUE #743: Add logging level argument to cli
To allow users to quickly debug something, the logging level should be configurable from the cli.

### PR DESCRIPTION
**Pull request by twentyse7en**
Logging level can be configured from cmd line, default INFO.
fix #743 

## Contributor Checklist:

* [x] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
* [x] I have added/updated documentation for all user-facing changes

### CODE REVIEW

**Comment by twentyse7en** [comment]
Alias of `--local-directory` is changed to `-d`,  so `-l` can be used for `--log-level`. Is this ok @sk1p ?

**Comment by sk1p** [comment]
> Alias of `--local-directory` is changed to `-d`,  so `-l` can be used for `--log-level`. Is this ok @sk1p ?

Yep! 👍

**Comment by twentyse7en** [comment]
is there anything else to do? @sk1p

File: src/libertem/web/server.py [hunk 1]
```diff
@@ -119,9 +119,12 @@ def sig_exit(signum, frame, shared_state):
     )
 
 
-def main(host, port, event_registry, shared_state):
+def main(host, port, log_level, event_registry, shared_state):
+    numeric_level = getattr(logging, log_level.upper(), None)
+    if not isinstance(numeric_level, int):
+        raise ValueError(f'Invalid log level: {log_level}')
```
Comment (sk1p) [inline src/libertem/web/server.py:125]: I would prefer to have this validation logic in the CLI part instead. Then, you can [raise a `click.UsageError` instead](https://github.com/LiberTEM/LiberTEM/blob/b7c1b148727eb5bb89ee00a16b35be3901694d87/scripts/release#L545), which should give a better user experience than just printing a Traceback.

File: src/libertem/web/cli.py [hunk 2]
```diff
@@ -4,17 +4,18 @@
 @click.command()
 @click.option('-p', '--port', help='port on which the server should listen on',
               default=9000, type=int)
-@click.option('-l', '--local-directory', help='local directory to manage dask-worker-space files',
+@click.option('-d', '--local-directory', help='local directory to manage dask-worker-space files',
               default='dask-worker-space', type=str)
 @click.option('-b/-n', '--browser/--no-browser',
               help='enable/disable opening the browser', default='True')
+@click.option('-l', '--log-level', help='set logging level', default='INFO')
```
Comment (sk1p) [inline src/libertem/web/cli.py:11]: The help message should maybe list the available logging levels (excluding `NOTSET`), so people can use this without having to look up the levels in the Python documentation.

**Comment by sk1p** [comment]
Sorry, didn't want to approve actually - read it as a "functionality approved, usability should be improved", please :grin:

**Comment by twentyse7en** [comment]
Thx for the review : ) will fix ASAP :+1:

File: src/libertem/web/cli.py [hunk 3]
```diff
@@ -1,20 +1,28 @@
 import click
+import logging
+
+log_values = "Allowed values are 'critical', 'error', 'warning', 'info', 'debug'."
```
Comment (twentyse7en) [inline src/libertem/web/cli.py:4]: Adding the available levels in help was a bit lengthy, also it can be reused with the error msg.

**Comment by sk1p** [comment]
Nice, thanks! :sparkles:

**Pull request merged**